app/views/users/_vitals.html.erb

```diff
@@ -0,0 +1,3 @@
+        <i class="glyphicon glyphicon-time"></i> Joined on <%= @user.created_at.to_date.strftime("%b %d, %Y") %> <br />
+        <i class="glyphicon glyphicon-folder-open"></i> Deposited Files
+        <%= link_to number_of_deposits(@user), search_action_path(render_facet_params("depositor_ssim", @user.to_s)) %> <br />
```

Comment:
@pgwillia definitely more straight forward, didn't see this method but I will change it.